### ISSUE TITLE
paper

### ISSUE THREAD

**Issue by aliechoes** [saeyslab/DeepLearning_for_ImagingFlowCytometry#4]
Hi

For which paper this package is developed?

**Comment by MaximLippeveld** [comment]
Hi, this for the paper [Classification of Human White Blood Cells Using Machine Learning for Stain‐Free Imaging Flow Cytometry](https://doi.org/10.1002/cyto.a.23920). Thanks for your interest!

**Comment by aliechoes** [comment]
thanks for the reply. I actually found your paper quite interesting. I am also working on IFC as well. so it was a quite good to see your work.
cheers

**Comment by aliechoes** [comment]
Thanks again for the reply. I found the paper quite interesting. I am wondering whether the data is publicly available for experiments?

**Comment by MaximLippeveld** [comment]
Hi, glad you liked it! The data is available. [This directory](https://cloud.irc.ugent.be/public/index.php/s/assnP3Z2FjTbztc) contains everything used in the paper.

**Comment by aliechoes** [comment]
awesome! thanks :)

**Comment by aliechoes** [comment]
Hi Maxim, So I download the files and try to use `cifconvert`. However it seemed quite challenging to convert the files. Also, I was not able to open the `.daf` files using IDEAS. Not sure what was the problem. 
I am wondering if the `hdf5` or `.tif` files are somewhere stored which I can use directly? 
in addition, can you please specificy which channels are used? Thanks a lot!

**Comment by aliechoes** [comment]
also, which channels were used in the analysis?

**Comment by MaximLippeveld** [comment]
Indeed, cifconvert isn't bug free, unfortunately :-) It should work in most caes, though. What problems did you run into?

What version of IDEAS are you using to open the daf files?

I don't have the hdf5 files available, currently. Can you open the cifs in IDEAS? Then you can export the tif files, maybe.

**Comment by aliechoes** [comment]
I am using IDEAS 6.2. It tells that there is something wrong with the .daf files though. 
I could however open the .cif file alone. but I was not sure if all the cells in the .cif files are the one used in your work. For example, I was not sure whether focused cells were gated or not.

**Comment by MaximLippeveld** [comment]
I indeed used only single, focused cells. Have you tried re-downloading the .daf files? I will try later today whether I can still open them.
